src/lib/components/consignments/__stories__/consignments.story.tsx

```diff
@@ -0,0 +1,17 @@
+import { storiesOf } from "@kadira/react-native-storybook"
+import * as React from "react"
+import { RootContainer } from "react-relay"
+
+import Info from "../setup/info"
+import Welcome from "../setup/welcome"
+
+const nav = {} as any
+const route = {} as any
+
+storiesOf("Consignments")
+  .add("Welcome Page", () => {
+    return <Welcome navigator={nav} route={route} />
+  })
+  .add("Info Page", () => {
+    return <Info navigator={nav} route={route} />
+  })
```

Comment:
How do you feel about doing `.add("Info Page", () => <Info navigator={nav} route={route} />)`? Is it too much optimization?